.github/workflows/ci.yml

```diff
@@ -47,6 +52,10 @@ jobs:
 
   build-ubuntu-latest-full-featured-gcc:
     runs-on: ubuntu-latest
+    # Enable LTO, might trigger additional warnings on advanced inlining
+    env:
+      CFLAGS: -O3 -g -flto
+      LDFLAGS: -flto
```

Comment:
With LTO you should repeat the flags for `-O3 -g` to ensure they appear in the final linking command.

Reply:
added